src/module-1/splitAndMerge/index.js

```diff
@@ -1 +1,10 @@
-export const splitAndMerge = (str = "", separator = "") => {};
+export const splitAndMerge = (str = "", separator = "") => {
+  let splitStr =  str.split(" ");
```

Comment:
```suggestion
  const words =  str.split(" ");
```  
major: use consts where the value is not supposed to be changed  
minor: consider to rename this to words